components/FlexibleForms/ComboboxField.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
```

Comment:
This would be a lot easier to review if a storybook component was included.

It's fairly complex to review this code without seeing it in action somewhere.

Reply:
i agree. are we anywhere with working out that storybook bug? i still can't run it on my computer. i can dig into it but can't estimate how long it will take